android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/harmony/TestHarmonyTransactionSigner.kt

```diff
@@ -0,0 +1,39 @@
+package com.trustwallet.core.app.blockchains.harmony
+
+import com.google.protobuf.ByteString
+import com.trustwallet.core.app.utils.toHexByteArray
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import wallet.core.jni.PrivateKey
+import wallet.core.jni.HarmonySigner
+import wallet.core.jni.proto.Harmony
+import com.trustwallet.core.app.utils.Numeric
+
+class TestHarmonyTransactionSigner {
+
+    init {
+        System.loadLibrary("TrustWalletCore")
+    }
+
+    @Test
+    fun testHarmonyTransactionSigning() {
+        val signingInput = Harmony.SigningInput.newBuilder()
+        signingInput.apply {
+            privateKey = ByteString.copyFrom(PrivateKey("0xe2f88b4974ae763ca1c2db49218802c2e441293a09eaa9ab681779e05d1b7b94".toHexByteArray()).data())
+            toAddress = "one1tp7xdd9ewwnmyvws96au0e7e7mz6f8hjqr3g3p"
+            chainId = ByteString.copyFrom("0x190".toHexByteArray())
+            nonce = ByteString.copyFrom("0x9".toHexByteArray())
+            gasPrice = ByteString.copyFrom("0x0".toHexByteArray())
+            gasLimit = ByteString.copyFrom("0x5208".toHexByteArray())
+            fromShardId = ByteString.copyFrom("0x3".toHexByteArray())
+            toShardId = ByteString.copyFrom("0x2".toHexByteArray())
+            amount = ByteString.copyFrom("0x168d28e3f00280000".toHexByteArray())
+        }
+
+        val sign: Harmony.SigningOutput = HarmonySigner.sign(signingInput.build())
+
+        assertEquals(Numeric.toHexString(sign.v.toByteArray()), "0x0343")
```

Comment:
where did you get the golden value 0x0343 from ?  Is it from harmony go output ?